events/events.md

```diff
@@ -5,40 +5,36 @@ title: Events
 description: Machine translation events
 ---
 
-## Upcoming events
-
-| Date | Event | Organisation |
+### 2022
+| Date | Event | Location |
 | ---- | ---- | ---- |
-| May 26, 2022 | [IWSLT 2022](iwslt2022.md) | [SIGSLT](/associations/SIGSLT.md) |
-| Jun 01, 2022 | [EAMT 2022](eamt2022.md) |  [EAMT](/associations/eamt.md)  |
-| Jul 02, 2022 | [NeTTT Conference](nettt2022.md) | [NeTTT Organisers](nettt2022.md#Organisers) |
-| Sep 12, 2022 | [AMTA 2022](amta2022.md) | [AMTA](/associations/amta.md) |
-| Oct 12, 2022 | [LoResMT 2022](loresmt2022.md) | LoResMT |
-
-
-## Planned events
-
-| Date | Event |
-| ---- | ---- |
-| 2022 | WMT22 |
-| 2022 | AAMT 2022 |
-| 2023 | MT Summit 2023 |
-
-
-## Past events
+| November, 2022 | [**WMT22**](wmt22.md) | |
+| 2022 | AAMT 2022 | |
```

Comment:
Kozo from AAMT mentioned that AAMT 2022 will probably take place in early December.